specs/swagger.yml

```diff
@@ -1,11 +1,11 @@
 ---
 swagger: '2.0'
 info:
```

Comment:
Would it be worth looking into updating this automatically as part of the build? Not in the context of this PR, but for future work..

Reply:
Right now the version tracks the Ziti release number but what are we going to do when we have v1, v2, ...vN REST API?

We could do something like `v1.0.19.0` `v2.0.19.0` or such.

Reply:
hmm, that's interesting. Lots of options... We could keep them at the same version, but separate them by title. We could version them completely independently and only update the versions when the swagger file actually changes. That would probably be the most helpful to users, but the most work for us. We could also update a version in a comment or the title so that people could see what version of ziti it's associated with.